osv/sources.py

```diff
@@ -27,13 +29,16 @@
 # pylint: disable=relative-beyond-top-level
 from . import repos
 from . import vulnerability_pb2
+from . import cache
 
 AUTHOR_EMAIL = 'infra@osv.dev'
 PUSH_RETRIES = 2
 PUSH_RETRY_SLEEP_SECONDS = 10
 
 YAML_EXTENSIONS = ('.yaml', '.yml')
-JSON_EXTENSIONS = ('.json')
+JSON_EXTENSIONS = '.json'
```

Comment:
I think the extension was for this to be a tuple (which it previously was not because of a missing ,).

Can you make this `('.json.',)` instead?